batcher/aligned/src/main.rs

```diff
@@ -102,6 +110,37 @@ pub struct SubmitArgs {
     keystore_path: Option<PathBuf>,
 }
 
+#[derive(Parser, Debug)]
+#[command(version, about, long_about = None)]
+pub struct DepositToBatcherArgs {
+    #[arg(
+        name = "Batcher Eth Address",
+        long = "conn",
+        default_value = "0x7969c5eD335650692Bc04293B07F5BF2e7A673C0"
+    )]
+    batcher_eth_address: String,
+    #[arg(
+        name = "Path to local keystore",
+        long = "keystore_path",
+        required = true
+    )]
+    keystore_path: Option<PathBuf>,
+    #[arg(
+        name = "Ethereum RPC provider address",
+        long = "rpc",
```

Comment:
should we match `cast --rpc-url` flag format?
```suggestion
        long = "rpc-url",
```

Reply:
It is the same convention in the other commands, I agree tho